Packs/MicrosoftEntraID/ParsingRules/MicrosoftEntraID/MicrosoftEntraID.xif

```diff
@@ -1,5 +1,5 @@
 [INGEST:vendor="msft", product="azure", target_dataset="msft_azure_raw", no_hit=drop]
-filter category not in("AZFW*", "ApplicationGateway*", "AppServiceHTTPLogs", "AppServiceConsoleLogs", "AppServiceAppLogs", "AppServiceIPSecAuditLogs", "AppServicePlatformLogs", "AppServiceAntivirusScanAuditLogs", "AppServiceFileAuditLogs", "FunctionAppLogs", "AppServiceAuditLogs", "WorkflowRuntime", "AppServiceEnvironmentPlatformLogs", "cloud-controller-manager", "cluster-autoscaler", "kube-audit", "kube-audit-admin", "kube-apiserver", "kube-controller-manager", "kube-scheduler", "csi-snapshot-controller", "csi-azuredisk-controller", "csi-azurefile-controller", "SoftwareUpdateProfile", "SoftwareUpdates", "guard")
+filter category not in("AZFW*", "ApplicationGateway*", "AppServiceHTTPLogs", "AppServiceConsoleLogs", "AppServiceAppLogs", "AppServiceIPSecAuditLogs", "AppServicePlatformLogs", "AppServiceAntivirusScanAuditLogs", "AppServiceFileAuditLogs", "FunctionAppLogs", "AppServiceAuditLogs", "WorkflowRuntime", "AppServiceEnvironmentPlatformLogs", "cloud-controller-manager", "cluster-autoscaler", "kube-audit", "kube-audit-admin", "kube-apiserver", "kube-controller-manager", "kube-scheduler", "csi-snapshot-controller", "csi-azuredisk-controller", "csi-azurefile-controller", "SoftwareUpdateProfile", "SoftwareUpdates", "guard") AND OperationName NOT IN ("Artifacts.Feed.*", "AuditLog.AccessLog", "AuditLog.DownloadLog", "AuditLog.Stream*", "AuditLog.TestStream", "Billing.*", "Extension.*", "Git.*", "Group.*", "Library.*", "Licensing.*", "Organization.*", "OrganizationPolicy.*", "Pipelines.*", "Policy.PolicyConfig*", "Process.Behavior.*", "Process.Control.*", "Process.Field.*", "Process.Group.*", "Process.List.*", "Process.Page.*", "Process.Process.*", "Process.Rule.*", "Process.State.*", "Process.SystemControl.*", "Process.WorkItemType.*", "Project.AreaPath.*", "Project.Create*", "Project.Delete*", "Project.HardDelete*", "Project.Restore*", "Project.SoftDelete*", "Project.UpdateRename*", "Project.UpdateVisibility*", "Project.IterationPath.*", "Project.Process.*", "Release.*", "Security.*", "Token.*")
```

Comment:
Won't adding just-
**Type = "AzureDevOpsAuditing"**
For the exclusion would be better?